src/models/Order.js

```diff
@@ -0,0 +1,49 @@
+import { Schema, model } from "mongoose";
+
+const conectionString =
+  "mongodb+srv://pftestuser:dCIJSIvnkiwSUaug@cluster0.rzccgck.mongodb.net/?retryWrites=true&w=majority";
+
+const orderSchema = new Schema(
+  {
+    userID: {
+      type: String,
+      required: true,
+    },
+    totalAmount: {
+      type: Number,
+      required: true,
+    },
+
+    itemName: {
```

Comment:
itemName, itemQuantity and itemPrice should be replaced by the same products array from Cart schema using reference